src/backend/trace.rs

```diff
@@ -410,12 +410,12 @@ impl TraceData {
                 hop.total_time += dur;
                 // Before last is set use it to calc jitter
                 let last_ms = hop.last_ms().unwrap_or_default();
-                let jitter_ms = (last_ms - dur_ms).abs();
+                let jitter_ms = (dur_ms - last_ms).abs();
```

Comment:
@trkelly23 not sure if we need this change, it is the same either way since we take the absolute value.  What is the rationale here?

Reply:
I found a slight difference in the spec'd math of RFC3550 and changed it to align the code with the RFC.

Reply:
ok makes sense!